src/category_theory/limits/cones.lean

```diff
@@ -417,48 +362,35 @@ namespace category_theory.limits
 
 variables {F : J ⥤ Cᵒᵖ}
 
-def cone_of_cocone_left_op (c : cocone F.left_op) : cone F :=
+@[simps] def cone_of_cocone_left_op (c : cocone F.left_op) : cone F :=
```

Comment:
I'm not sure this is a good idea, because now there are two simp lemmas for the `π` field and it's not clear to me whether they are confluent. If they are, then we should probably delete `cone_of_cocone_left_op_π_app`.

Reply:
I agree, and will revert here.